test/models/concerns/businesses/hiring_invoice_requests/notifications_test.rb

```diff
@@ -0,0 +1,23 @@
+module Businesses
+  module HiringInvoiceRequests
+    class Notification::NotificationsTest < ActiveSupport::TestCase
+      include ActionMailer::TestHelper
```

Comment:
```suggestion
```suggestion
require "test_helper"

module Businesses
  module HiringInvoiceRequests
    class NotificationsTest < ActiveSupport::TestCase
```

Reply:
added in d630a6dbcf93efbead42ba91758d5e614891d55a